src/components/header/header.tsx

```diff
@@ -5,24 +5,24 @@ import { useApi } from "../../hooks/use-api";
 import { PersonTotalTime } from "../../generated/client";
 import { errorAtom } from "../../atoms/error";
 import LoaderWrapper from "../generics/loader-wrapper";
-import {  Container, Grid } from "@mui/material";
+import { Container, Grid } from "@mui/material";
 import BalanceCard from "./balance-card";
 import HomeNav from "./nav";
 
 /**
  * Dashboard screen component
  */
-function Header () {
+const Header = () => {
```

Comment:
Looks like the JSDocs for this are outdated?? This is not the dashboard/ home screen

Reply:
fixed